less/variables.less

```diff
@@ -24,3 +24,4 @@
 @border-real: #dee2e6;
 
 // Mobile var
+@mobile: ~"(max-width: 500px)";
```

Comment:
-  Create a @phone variable that contains a max-width: 500px media query string. - DONE! 

- Use the @phone variable for all your nested mobile styling. - DONE!